kong/plugins/jwt/asn_sequence.lua

```diff
@@ -0,0 +1,82 @@
+local error = error
+
+local _M = {}
+_M.__index = _M
+
+function _M.create_simple_sequence(input)
+  if type(input) ~= "table" then error("Argument #1 must be a table", 2) end
+  local sortTable = {}
```

Comment:
There is a lot of error checking done. I suggest dropping this. Just verify external inputs (from the plugin configuration, or an incoming request). And stop there.

Add testcases for bad input and make sure the proper return codes are generated without leaking sensitive information.

in general, for bad input/user errors; return `nil + errormsg`, for hard technical errors, use `error` or `assert` (the latter two will mostly generate 500's, whereas for the former you'd want 400's)